### ISSUE TITLE
AirIAM adding functionality to scan more than one account

### ISSUE THREAD

**Issue by ali0818** [bridgecrewio/AirIAM#87]
Can we have a function or argument to scan more than one AWS account at a time or individual, as of now it only scan the one in AWS config, but it will be awesome if we can scan choosing the AWS config profiles?

**Comment by nimrodkor** [comment]
Hey @ali0818 !

Just wondering here - AirIAM is not a very fast tool, so running accounts sequentially can be quite painful. Wouldn't it make more sense to run them in parallel, via multiple processes for example?

**Comment by ali0818** [comment]
@nimrodkor yes, or at least if we can just choose multiple accounts and scan 1 from that. It would be awesome if AirIAM can check all accounts and prompt to choose which one to scan or if scan all account then running in parallel with a thread will be great.
